index.js

```diff
@@ -531,6 +523,12 @@ export default async function getCSS({
 				}],
 			});
 		}
+
+		rules.unshift({
+			type: 'rule',
+			selectors: ['.markdown-body'],
+			declarations: sharedDeclarations,
+		});
```

Comment:
I don't think one-theme-only styles needs these variables.

```suggestion

			rules.unshift({
				type: 'rule',
				selectors: ['.markdown-body'],
				declarations: sharedDeclarations,
			});
		}
```

Reply:
However, if moved like above, `dist/vars/dark.css` will miss these variables.